ga4gh/datamodel/datasets.py

```diff
@@ -210,11 +210,10 @@ def __init__(self, localId, dataDir, backend):
                     self, localId, relativePath, backend)
                 self.addVariantSet(variantSet)
             # Variant annotations sets
-            variantAnnotationSetDir = os.path.join(
-                relativePath, "variantAnnotations")
-            if os.path.isdir(variantAnnotationSetDir):
+            if self.isAnnotated(relativePath, localId):
                 variantAnnotationSet = variants.HtslibVariantAnnotationSet(
-                    self, localId, variantAnnotationSetDir, backend)
+                        self, localId, relativePath, backend,
```

Comment:
I think this is over-indented for hanging indent --- newer versions of pep8 might choke on this.